pydjamodb/test_runner.py

```diff
@@ -96,7 +96,7 @@ class DynamoDBTestSuiteMixin:
 
     parallel_test_suite = DynamoDBParallelTestSuite
 
-    def log(self, msg):
+    def log(self, msg, level=None):
```

Comment:
Myslím, že je tohle ok řešení. Je to převážně as-is stav, jen od Django 4 přibylo pár logů s debug levelem, ale nic dramatického. Kdyby začala být konzole nepřehledná, můžeme začít brát ty levely v potaz.